tests/common/dnspolicy/dnspolicy_controller_single_cluster_test.go

```diff
@@ -97,12 +96,6 @@ var _ = Describe("DNSPolicy Single Cluster", func() {
 				},
 			}
 			gateway.Status.Listeners = []gatewayapiv1.ListenerStatus{
-				{
-					Name:           "foo",
-					SupportedKinds: []gatewayapiv1.RouteGroupKind{},
-					AttachedRoutes: 0,
-					Conditions:     []metav1.Condition{},
-				},
```

Comment:
This was added for a reason https://github.com/Kuadrant/kuadrant-operator/pull/655/files#r1611332705

Reply:
It has no effect now. No listeners -> no DNS Records created -> no effect on the status of the policy as long as there are other records. It was `partially enforced` for the wrong reason.

Reply:
There is now a different [test](https://github.com/Kuadrant/kuadrant-operator/pull/722/files#diff-76eb28eef00ca4741889476a78254e8513eb9407be3a095fcee80a360f06a58fR403-R407) for the `PartiallyEnforced` case

Reply:
This was nothing to do with enforced. It was ensuring that we had a listener with 0 attached routes which before that PR was broken. Adding that was ensuring we had covered that scenario in our tests so we didn't break it again. i.e if i reverted [this](https://github.com/Kuadrant/kuadrant-operator/pull/655/files#diff-ad34d851ae1426ca68b2179d94856acd67b5af507fcd3b3ff275cd8121e53db6R93) for some reason, the tests would fail.

Reply:
Hm, i did forget about that bug. Brought it back. Now it will be checking, at least, for the correct status